ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineGetBlobsV1.java

```diff
@@ -0,0 +1,155 @@
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.methods.engine;
+
+import org.hyperledger.besu.datatypes.BlobsWithCommitments;
+import org.hyperledger.besu.datatypes.Transaction;
+import org.hyperledger.besu.datatypes.VersionedHash;
+import org.hyperledger.besu.ethereum.ProtocolContext;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.JsonRpcRequestContext;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.exception.InvalidJsonRpcParameters;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.methods.ExecutionEngineJsonRpcMethod;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.parameters.JsonRpcParameter;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.JsonRpcErrorResponse;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.JsonRpcResponse;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.JsonRpcSuccessResponse;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.RpcErrorType;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.results.BlobAndProofV1;
+import org.hyperledger.besu.ethereum.eth.transactions.BlobCache;
+import org.hyperledger.besu.ethereum.eth.transactions.TransactionPool;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import io.vertx.core.Vertx;
+
+/**
+ * #### Specification
+ *
+ * <p>1. Given an array of blob versioned hashes client software **MUST** respond with an array of
+ * `BlobAndProofV1` objects with matching versioned hashes, respecting the order of versioned hashes
+ * in the input array.
+ *
+ * <p>2. Client software **MUST** place responses in the order given in the request, using `null`
+ * for any missing blobs. For instance, if the request is `[A_versioned_hash, B_versioned_hash,
+ * C_versioned_hash]` and client software has data for blobs `A` and `C`, but doesn't have data for
+ * `B`, the response **MUST** be `[A, null, C]`.
+ *
+ * <p>3. Client software **MUST** support request sizes of at least 128 blob versioned hashes. The
+ * client **MUST** return `-38004: Too large request` error if the number of requested blobs is too
+ * large.
+ *
+ * <p>4. Client software **MAY** return an array of all `null` entries if syncing or otherwise
+ * unable to serve blob pool data.
+ *
+ * <p>5. Callers **MUST** consider that execution layer clients may prune old blobs from their pool,
+ * and will respond with `null` if a blob has been pruned.
+ */
+public class EngineGetBlobsV1 extends ExecutionEngineJsonRpcMethod {
+
+  private final Map<VersionedHash, BlobsWithCommitments.BlobQuad> blobMap = new HashMap<>();
```

Comment:
I think naming this something more specific would help me understand why it is necessary. As is, it reads like a synonym for blobCache, and seems like duplication.